processor/attributekeyprocessor/config.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attributekeyprocessor
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+)
+
+// Config defines configuration for Attribute Key processor.
+type Config struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+	// map key is the attribute key to be replaced.
+	KeyReplacements map[string]NewKeyProperties `mapstructure:"key-replacements"`
```

Comment:
in light of the syntax having the map instead of the old slice, simply naming this one `keys` seems better

Reply:
just to be clear I mean `keys` instead of `key-replacements`